pkg/asset/machines/master.go

```diff
@@ -606,3 +609,31 @@ func IsMachineManifest(file *asset.File) bool {
 		return matched
 	}
 }
+
+func generateAsset(objects interface{}, fileName string) (assets []*asset.File, err error) {
```

Comment:
Do not overload the term asset, which has a specific meaning in the context of the installer.
```suggestion
func generateAssetFile(objects interface{}, fileName string) (assets []*asset.File, err error) {
```